packages/expo-cli/src/commands/utils/validateDependenciesVersions.ts

```diff
@@ -71,10 +75,39 @@ export async function validateDependenciesVersionsAsync(
           )}`
       );
     }
+  }
 
-    return false;
+  if (Versions.gteSdkVersion(exp, '45.0.0')) {
```

Comment:
let's do this for exactly 45.0.0

Reply:
xdl doesn't expose an exact compare, and I don't want to duplicate the rest of the logic from that function (check for valid sdkVersion in json, compare, handle errors etc.) for such a rudimentary check.
```suggestion
  if (Versions.gteSdkVersion(exp, '45.0.0') && !Versions.gteSdkVersion(exp, '46.0.0')) {
```
Would this be an acceptable alternative?

Reply:
yeah that's fine - although i would have a slight preference for extending `Versions` to add `Versions.equalsSdkVersion`